### DISCUSSION TITLE
Upgraded VPS to Debian Buster and now Teslamate is wonky

### DISCUSSION THREAD

**Discussion by cyberlauncher** [teslamate-org/teslamate#1608] [Q&A] [answered]
So...
Like the title says, I updated my VPS from Debian Stretch to Buster (10) and now my Docker version of Teslamate is hosed.
And it was working pretty-much perfectly for the last several months collecting drive data.
Now, I simply can't login to the main page from the internet. Basically "site is unreachable" with a timeout.
When tried from the server itself using "localhost:4000" as the address I get a "secure connection failed"
Startup seems normal. Docker-compose logs seem ok with repeating "waiting for postgres at db:5432".
Several restarts had no effect.
Netstat shows that docker-proxy is listening to port 4000.
The current docker-compose version is 1.21.1
I don't remember what the previous instance was.
Not sure what the hangup is.
I had the presence of mind to make another backup of the Teslamate db beforehand.

Any clues?

**Comment by DrMichael** [comment, marked as answer]
> Docker-compose logs seem ok with repeating "waiting for postgres at db:5432".

Looks as if the database is not up / reachable?

> Netstat shows that docker-proxy is listening to port 4000.

what about port 5432? What says `docker-compose ps`?

> The current docker-compose version is 1.21.1

Yes, I have TM running under buster with that version.

**Reply by cyberlauncher** [reply]
Thank you for your response but the issue has been resolved.
I'm unsure exactly what had affected TM but after trying another docker container, it (seemly) fix whatever the issue was with TM.
Curious.

**Reply by DrMichael** [reply]
Then you could close here?
